### DISCUSSION TITLE
Discussion Policies

### DISCUSSION THREAD

**Discussion by MetzinAround** [projectdiscovery/nuclei#3602] [General]
We're updating some policies around discussions! For discussions in this space we'll have a few new processes. 

- Discussions older than 6 months are going to be closed. 
- Discussions with a marked answer will be closed. 
- Whenever Community interacts with a discussion by posting a comment, the label "Community ✅" will be applied. 
- We will move or relabel discussions as appropriate for what the OP is trying to accomplish. 

Thank you for being a valued community member and we look forward to continuing to work with you all here on GitHub!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
